department-of-veterans-affairs/va.gov-cms
# VACMS-2015: allow content admins to browse sections from their profiles

## Description

See #2015  

## Testing done

Added behat test to confirm that content admins see the section browse text and don't see the "You don't have permission to access content in any VA.gov sections yet." when viewing their profile.

## Screenshots


## QA steps

As a user with the "content admin" role,
1. Go to the user profile page (/user)
1. Ensure that you see the text "You can edit content in the following VA.gov sections.", along with a table of sections.
- [ ] Correct text and sections are present.
1. Then validate Acceptance Criteria from issue

## Definition of Done
